### PR TITLE
Use Pimlico RPC for fee estimation

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
doesn't solve low gas issue yet (https://basescan.org/tx/0x476163c19a5456b02034fd2b8f4488256c66d48c18f9bc9657e76049be286ead)

### CODE REVIEW

File: packages/daimo-api/src/network/bundlerClient.ts [hunk 1]
```diff
@@ -37,6 +48,17 @@ export class BundlerClient {
...
+
+  async getUserOperationGasPriceParams() {
+    const gasPrice = (await this.provider.send(
+      "pimlico_getUserOperationGasPrice",
+      []
+    )) as GasPrice;
+    console.log(
+      `[BUNDLER] fetched gas price params: ${JSON.stringify(gasPrice)}`
+    );
+    return gasPrice.fast;
```
Comment (dcposch) [inline packages/daimo-api/src/network/bundlerClient.ts:60]: 👍

**Review by dcposch — APPROVED** [review]
LGTM

seems like the root cause is pimlico returning too low preVerificationGas?

**Comment by nalinbhardwaj** [comment]
> seems like the root cause is pimlico returning too low preVerificationGas?

preVerificationGas looks correctly set to me. maxFeePerGas and priority fee are lower by ~10x than what the bundler actually sets afaict? (and that diff remains true even after this change)

**Comment by dcposch** [comment]
cherry-picked into #451

**Pull request closed without merging**